deployment/flatpak/usage.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Flatpak Usage
+subsection: flatpak
+section: deployment
```

Comment:
You need to add order to all pages. Example:
https://github.com/developer-portal/content/blame/master/tech/languages/haskell/haskell-installation.md#L5